backend/src/main/java/controllers/PostController.java

```diff
@@ -69,23 +69,18 @@ protected void browsePost(Post selectedPost) {
      */
     private String getCountableIngredients(String promptCountable) {
         String countableInput = this.inOut.getInput(promptCountable);
-        if (countableInput.contains("N/A") && this.noMeasurableIngredients) {
+        boolean empty = countableInput.contains("N/A") || countableInput.strip().equals("");
+        if (empty && this.noMeasurableIngredients) {
             inOut.setOutput("Must enter at least one ingredient");
             countableInput = getCountableIngredients(promptCountable);
-        } else if (countableInput.contains("N/A")) {
+        } else if (empty) {
             return "N/A";
         }
         String[] splitIngredients = countableInput.split(",");
-        String errorMessage = "That was an invalid input. Please enter countable ingredients again in correct format.";
+        String errorMessage = "That was an invalid input. Please enter ingredients again in correct format.";
         for (String ingredient: splitIngredients) {
             String strippedIngredient = ingredient.stripLeading();
-            if (!Character.isDigit(strippedIngredient.charAt(0))) {
-                inOut.setOutput("Invalid input. Did not specify numerical amount of one of the ingredients. Please enter countable ingredients again in correct format.");
-                countableInput = getCountableIngredients(promptCountable);
-            } else if (strippedIngredient.split(" ").length < 2) {
-                inOut.setOutput(errorMessage);
-                countableInput = getCountableIngredients(promptCountable);
-            } else if (strippedIngredient.contains("/")) {
+            if (checkInputInvalidity(strippedIngredient) || strippedIngredient.split(" ").length < 2) {
```

Comment:
I think a better name would be "isInputInvalid" to reflect the boolean return value

Reply:
great point